skyportal/models/tag.py

```diff
@@ -14,5 +12,22 @@ class ObjTagOption(Base):
     tag_name = sa.Column(sa.String, nullable=False, doc="Tags list.", unique=True)
 
 
-ObjTags = join_model("obj_tags", ObjTagOption, Source)
+ObjTags = join_model("obj_tags", ObjTagOption, Obj)
+
+read = update = delete = AccessibleIfUserMatches("user")
+
+ObjTags.user_id = sa.Column(
```

Comment:
I'm fine with that, though do keep in mind that for most tables where a user is associated to the record and it's somewhat text related (comment, classification, ...) we usually call it `author_id`. Consistency in coding is good in general because when working on any feature, if its inner workings and field names are similar to other similar concepts, it's easier for the developer.